ref_ts/views.py

```diff
@@ -221,7 +221,7 @@ def download_refts_resource_files(request, shortkey, *args, **kwargs):
             return response
 
         path = "bags/" + str(shortkey) + ".zip"
-        response_irods = download_bag_from_irods(request, path)
+        response_irods = download_bag_from_irods(request, path, use_async=False)
```

Comment:
@hyi Since this line from REST API (https://github.com/hydroshare/hydroshare/blob/develop/hs_core/views/resource_rest_api.py#L271)redirects to this view function, we are missing the setting for ```rest_call```.